app/src/main/java/org/p2p/wallet/common/date/ZonedDateExtensions.kt

```diff
@@ -51,5 +51,10 @@ fun Long.toZonedDateTime(): ZonedDateTime = ZonedDateTime.ofInstant(
     ZoneId.systemDefault()
 )
 
+fun String.toZonedDateTime(): ZonedDateTime = ZonedDateTime.ofInstant(
+    Instant.parse(this),
+    ZoneId.systemDefault()
```

Comment:
this may give you an exception and the app may crash.

Reply:
yes it throws DateTimeParseException – if the text cannot be parsed or 
DateTimeException – if the result exceeds the supported range. Maybe add try, if catch return ""?